opencga-core/src/main/java/org/opencb/opencga/core/models/project/Project.java

```diff
@@ -97,6 +98,9 @@ public class Project extends PrivateFields {
             description = FieldConstants.PROJECT_ORGANISM)
     private ProjectOrganism organism;
 
+    @DataField(id = "cellbase", indexed = true, uncommentedClasses = {"CellBaseConfiguration"},
+            description = FieldConstants.PROJECT_INTERNAL_CELLBASE)
```

Comment:
FieldConstants variable should now be called PROJECT_CELLBASE instead of PROJECT_INTERNAL_CELLBASE